### PR TITLE
#2 create project

### PR DESCRIPTION
**Pull request by KuzykY**
Set up development environment,initialize Django project,configure settings for PostgreSQL.

Add .env and .env.example.

### CODE REVIEW

**Comment by akostromin14** [comment]
Looks great to me! Why didn't you asign yourself and add the milestone? Maybe I'm wrong about milestone, then correct me)

**Comment by KuzykY** [comment]
> Looks great to me! Why didn't you asign yourself and add the milestone? Maybe I'm wrong about milestone, then correct me)

Because the mentor said add to reviewers one mentor and two teammates. And I randomly chose two teammates)

File: forum/configs/settings.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+# Database
+# https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ.get('POSTGRES_DB'),
+        'USER': os.environ.get('POSTGRES_USER'),
+        'PASSWORD': os.environ.get('POSTGRES_PASSWORD'),
+        'HOST': os.environ.get('POSTGRES_HOST'),
```
Comment (akostromin14) [inline forum/configs/settings.py:83]: To connect docker db with django, we should arrange with POSTGRES_HOST. In docker it should be named as container name.

Comment (akostromin14) [reply forum/configs/settings.py:83]: My proposal is to create an additional variable in .env called POSTGRES_DJANGO_HOST with the value of database container name. What do you think @VovaD199?

Comment (VovaD199) [reply forum/configs/settings.py:83]: I'm agree.

**Review by VovaD199 — APPROVED** [review]

**Pull request merged**